contributors/guide/README.md

```diff
@@ -247,7 +247,7 @@ If you're looking to run e2e tests on your own infrastructure, [kubetest](https:
 ## Issues Management or Triage
 
 Have you ever noticed the total number of [open issues](https://issues.k8s.io)?
-Helping to manage or triage these open issues can be a great contribution and a great opportunity to learn about the various areas of the project.
+Helping to manage or triage these open issues can be a great contribution and a great opportunity to learn about the various areas of the project.  Triaging is the word we use to describe the process of adding multiple types of descriptive labels to GitHub issues, in order to speed up routing issues to the right folks.  Refer to the [Issue Triage Guidelines](/contributors/guide/issue-triage.md) for more information.
```

Comment:
Can you remove `Refer to the [Issue Triage Guidelines](/contributors/guide/issue-triage.md) for more information.` part at the end? This line is already present below. Happy to add lgtm after this. :)